index.bs

```diff
@@ -951,7 +958,7 @@ Schedules the computational workload of a compiled {{MLGraph}} on the {{MLContex
     **Returns:** {{undefined}}.
 </div>
 
-Note: `dispatch()` itself provides no signal that graph execution has completed. Rather, callers should await the results of reading back the output tensors. See [[#api-mlcontext-dispatch-examples]] below.
+Note: `dispatch()` itself provides no signal that graph execution has completed. Rather, callers can `await` the results of reading back the output tensors. See [[#api-mlcontext-dispatch-examples]] below.
```

Comment:
🤔 Shouldn't this `should` still be a `should`? `Can` sounds overly permissive, as you really wouldn't want to read the output tensor data before it is actually available to be read (that is, read the `ArrayBuffer` before `mlContext.readTensor` finished).

Reply:
> often leads to some linguistic contortions

Indeed... 😉